rmw_dps_cpp/test/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+find_package(ament_cmake_gmock REQUIRED)
+
+ament_add_gmock(test_node
```

Comment:
You might want to put test_node in a list and do everything in a foreach loop, but this can easily be modified later so not a big deal.

Reply:
Agreed, I just wanted to get the infrastructure hooks in place with this commit for future unit tests.